src/lib/themes.js

```diff
@@ -125,8 +125,9 @@ export const normalizeThemeBackground = background =>
 
 // Utility to ensure normal & consistent colors
 export const normalizeThemeColor = (name, data, defaultColor) => {
-  const color = makeTinycolor(data || defaultColor).toRgb();
-  if (!colorsWithAlpha.includes(name)) {
+  const inColor = data || defaultColor;
+  const color = makeTinycolor(inColor).toRgb();
+  if (colorsWithoutAlpha.includes(name) || (inColor && !("a" in inColor))) {
```

Comment:
If alpha has the default value, it can also be deleted IMO.

Could you add unit tests to cover this new logic?

Reply:
I'm a bit confused. I initially proposed this.
![image](https://user-images.githubusercontent.com/922368/88566049-1f4e5b00-d03e-11ea-9a71-7eafb51cf061.png)
https://github.com/mozilla/FirefoxColor/pull/944#issuecomment-663971106
Then I implemented this change. https://github.com/mozilla/FirefoxColor/commit/b911023c3d60918866306d3b15b19a80ae7d6b88#diff-28b54e29959f783d6744df0f4e0c3c49R28
![image](https://user-images.githubusercontent.com/922368/88566127-40af4700-d03e-11ea-97d8-fdd57b30f970.png)
Then you told me to revert. https://github.com/mozilla/FirefoxColor/pull/944#issuecomment-664003245
![image](https://user-images.githubusercontent.com/922368/88566181-558bda80-d03e-11ea-9d09-24c8ebe0b25e.png)

I'm fine with either way but if we delete the opaque alpha, then the "[should not change preset themes](https://github.com/mozilla/FirefoxColor/blob/410e1eda9f6cc9a4837edefef136398712619c59/src/lib/themes-test.js#L27-L37)" will fail if we don't update the preset themes like I did before.
![image](https://user-images.githubusercontent.com/922368/88566377-95eb5880-d03e-11ea-8f1a-37f3f72855d9.png)

Am I missing some solution where we can delete the `a: 1` but we avoid updating the default themes?

Reply:
I thought that you deleted `"a": 1` in order to get it to pass the schema validation.

I'd be fine with removing `"a": 1` from the default theme templates if it is still backwards-compatible.
Could you verify that the website would still be compatible with the old add-on (i.e. the current 2.1.6) if you do remove `"a": 1`?

Reply:
do we get any benefit if we delete the a: 1?

Reply:
I suppose that the permalink to the URL would be smaller (i.e. the part after `?theme=...`).